app/assets/stylesheets/modules/_article_viewer.styl

```diff
@@ -153,3 +154,55 @@
   sup
     vertical-align super
     font-size smaller
+
+.wikibase-statementgroupview-property-label 
```

Comment:
Can the classes be a bit succinct while still conveying the meaning?

Reply:
Sorry. I thought you were adding these classes, whereas you're only styling the existing markup.